node_api/src/services/entity.service.ts

```diff
@@ -301,4 +301,37 @@ export default class EntityService {
       })
     ).data;
   }
+
+  async findEntitiesBySnomedCodes(codes: string[]) {
```

Comment:
suggest making new type (or extended of existing)containing iri, name,code,validationCode.